src/toString.js

```diff
@@ -0,0 +1,13 @@
+const toString = intake => {
+  let output = ""
+
+  if (intake === undefined){
```

Comment:
Since this is the same result as `intake === null`, you can combine into one boolean expression:
```es6
if( intake === undefined || intake === null ) {
  return output
}
```